docs_source_files/content/introduction/on_test_automation.es.md

```diff
@@ -4,166 +4,117 @@ weight: 2
 ---
 
 {{% notice info %}}
```

Comment:
@AlexAndradeSan could you please remove the “notice”?
The page has been fully translated, so it’s not needed anymore